modules/microshift-migrating-ipv6-dual-stack-config.adoc

```diff
@@ -0,0 +1,161 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-using-config-tools.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-migrating-ipv6-dual-stack-config_{context}"]
+= Migrating a {microshift-short} to IPv6 dual-stack networking
+
+You can convert your IPv4 single-stack cluster to dual-stack cluster networking that supports IPv4 and IPv6 address families by setting two entries in the service and cluster network parameters in the {microshift-short} configuration file.
+
+* The first IP family in the configuration is the primary IP stack in the cluster.
+* After the cluster is migrated to dual-stack networking and has restarted, enable pods and services for dual-stack networking by restarting them.
+
+[IMPORTANT]
+====
+The OVN-Kubernetes network plugin requires that both IPv4 and IPv6 default routes be on the same network device. IPv4 and IPv6 default routes on separate network devices is not supported.
+====
+
+[IMPORTANT]
+====
+When using dual-stack networking where IPv6 is required, you cannot use IPv4-mapped IPv6 addresses, such as `::FFFF:198.51.100.1`.
+====
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* Your cluster uses the OVN-Kubernetes network plugin.
+* The host has both IPv4 and IPv6 addresses and routes, including a default for each.
+* The host has at least two L3 networks, IPv4 and IPv6.
+
+.Procedure
+
+. If you have not done so, make a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, renaming it `config.yaml`.
+
+. Keep the new {microshift-short} `config.yaml` in the `/etc/microshift/` directory. Your `config.yaml` file is read every time the {microshift-short} service starts.
++
+[NOTE]
+====
+After you create it, the `config.yaml` file takes precedence over built-in settings.
+====
+
+. Add an IPv6 configuration for a second network in the `network` section of the {microshift-short} YAML with your desired valid values.
++
+.Example dual-stack `IPv6` configuration
+[source,yaml]
+----
+# ...
+apiServer:
+  subjectAltNames:
+  - 192.168.113.117
+network:
+  clusterNetwork:
+  - 10.42.0.0/16 <1>
+  serviceNetwork:
+  - 10.43.0.0/16
+node:
+  nodeIP: 192.168.113.117 <2>
```

Comment:
We need to add `nodeIPv6` here as well, this is a subtlety for dual stack.

Reply:
Would it also have a separate SAN?

Reply:
Yes. This is already looking good in the examples included in the PR :+1: